package.json

```diff
@@ -56,7 +63,7 @@
     "express": "^4.17.1",
     "husky": "^4.3.6",
     "lint-staged": "^10.5.3",
-    "parcel-bundler": "^1.12.4",
+    "parcel": "^2.0.0-beta.1",
```

Comment:
Should we use beta version?

Reply:
Parcel 1 seems to have not been released for a long time.
The v2 is the default branch of parcel, and it alpha version released in 2019.9.